src/main/java/com/thirdcc/webapp/repository/CourseProgramRepository.java

```diff
@@ -7,6 +7,8 @@
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
```

Comment:
why import this?

Reply:
I think I have added in accidently. Resolved in 3d3667fc1d901fb47691745754896ff1ea423842